### ISSUE TITLE
Import of Synthetic tests fails if CI option is defined in UI

### ISSUE THREAD

**Issue by ali-sattari** [DataDog/terraform-provider-datadog#1583]
Hello!

Once CI settings for a synthetic test is changed in the Datadog UI, the test cannot be imported in Terraform with following error:

```
% terraform import --var-file=local.tfvars datadog_synthetics_test.sample-test xxx-xxx-xxx
datadog_synthetics_test.sample-test: Importing from ID "xxx-xxx-xxx"...
datadog_synthetics_test.sample-test: Import prepared!
  Prepared datadog_synthetics_test for import
datadog_synthetics_test.sample-test: Refreshing state... [id=xxx-xxx-xxx]

Error: options_list.0.ci: '': source data must be an array or slice, got map

```

The API also lists CI under Options as map: [ref](https://docs.datadoghq.com/api/latest/synthetics/?code-lang=curl#get-a-browser-test). 

### Terraform Version

```
Terraform v1.2.5
+ provider registry.terraform.io/datadog/datadog v3.13.1
```

### Affected Resource(s)

-   datadog_synthetics_test

### Terraform Configuration Files

```hcl
resource "datadog_synthetics_test" "sample-test" {
  type = "browser"
  locations  = ["aws:eu-central-1"]
}
```

### Debug Output

[complete debug output](https://gist.github.com/ali-sattari/137909ec291814843d3858443e72a772)

### Panic Output

N/A

### Expected Behavior

Import should work fine with tests that have CI setting changed/defined (i.e. not null)

### Actual Behavior

Import gives error and fails.

### Steps to Reproduce

Refer to snippets above.

**Comment by nkzou** [comment]
I wasn't able to reproduce this bug - I created a synthetics browser test in the UI, changed the CI option execution rule from default "blocking" to "skipped" with the UI, and was able to successfully import it into a blank terraform state with your provided config snippet. Could you provide some more details about your setup/process?

**Comment by ali-sattari** [comment]
I tested again:
* Created a new test, no change to CI settings
* TF imported successfully using the basic resource block above into a blank state
* Removed the imported resource from TF state
* Changes the CI setting in UI to each of the values, and back to the default (Blocking)
* TF import fails after any change (regardless of the value) with above error

For reference, this is where/how I change CI settings, from Test Details page.
<img width="877" alt="Screenshot 2022-09-29 at 09 18 41" src="https://user-images.githubusercontent.com/122799/192966561-307c0e0d-d5bd-4690-834f-03dede40d2b5.png">


The only part I haven't included is this, but shouldn't make any difference:
```hcl
terraform {
  required_providers {
    datadog = {
      source = "DataDog/datadog"
    }
  }
}

provider "datadog" {
  api_url = "https://api.datadoghq.eu/"
  api_key = var.datadog_api_key
  app_key = var.datadog_app_key
}
```

**Comment by therve** [comment]
Can you try to upgrade your provider? Looks like it should have been fixed with #1491 which is in 3.14.0.

**Comment by ali-sattari** [comment]
Using v3.16.0 worked fine. Thanks @therve
